paasta_tools/cli/schemas/kubernetes_schema.json

```diff
@@ -700,6 +700,43 @@
                     },
                     "uniqueItems": true
                 },
+                "projected_volumes": {
```

Comment:
I think it will look similar to how they look on the pod specs themselves. For example, the [pod identity projected volume config on paasta contract monitor](https://fluffy.yelpcorp.com/i/vM9NmKsq666z5WCW7Z8mq31WS0B71wbg.html#L341-348,L429-440) looks like 

```
  volumes:
  - name: aws-iam-token
    projected:
      defaultMode: 420
      sources:
      - serviceAccountToken:
          audience: sts.amazonaws.com
          expirationSeconds: 86400
          path: token
```

Reply:
In paasta configs we'd have a simplified version of that looking like
```
projected_volumes:
- container_path: /var/whatever/something
   sources:
   - audience: a.b.c
     expiration_seconds: 1234
     path: token
```

Reply:
After the latest change
```yaml
projected_sa_volumes:
- container_path: /var/whatever/something
  audience: a.b.c
  expiration_seconds: 1234
```
much simpler

Reply:
just talking through the options:
* container_path: i assume this is configurable in case we need to mount this at a location that we can't configure due to some third-party dependency?
* audience: i haven't read through the entire CEP yet, but from where I did get to it sounds like this will be a static value? does this need to be user-configurable if so?
* expiration_seconds: i assume this is configurable since not everything that would consume this JWT would be ok with whatever the default expiration is?

Reply:
oh, I think i misread the bit in the CEP about audiences and whatnot: is the intent here that there could be multiple things that a given service would use a JWT for, and we'd have one mount for each of these things? (so that sensitive tokens could have short-lived lifetimes vs something that lets you query something like the ownership service could use the default expiration period?

that said: what would the downsides of having a single expiration value for everything be? if i'm reading the CEP correctly, I think mostly increased load on OPA?

Reply:
I'll try to answer all question together. As I mention in the PR description, I don't see this as the long term solution, I just want an interface in soa-configs which would allow to mount and automatically rotate a service account token, so that we can quickly iterate on the path / audience / expiration settings in the early stages of the project development.

In the long term I don't want developers to interface with this. I want everything to be abstracted by paasta-tools out of the box. I.e. on soa-configs changes, enumerate all services appearing in authorization policies, and mount projected SA volumes for all of them. If that's not going to be viable, then we'll have to settle in creating volumes for all services out of the box.

Reply:
perfect!